GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -59,6 +60,10 @@ namespace gdjs {
       // render the PIXI container of the scene
       this._pixiRenderer.backgroundColor = this._runtimeScene.getBackgroundColor();
       this._pixiRenderer.render(this._pixiContainer);
+      if (this._showCursor) {
```

Comment:
Nitpicking: let's add a line break and then a comment explaining why this operation is deferred there :)